5-Access-Authentication-and-Authorization/5.5-User-Accounts-and-Environment.bats

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bats
+
+load 5.5.4-helper
+load 5.5.5-helper
+
+@test "5.5.1.1 Ensure minimum days between password changes is configured (Automated)" {
+    run bash -c "grep PASS_MIN_DAYS /etc/login.defs | grep --invert-match \"\#\""
+    [ "$status" -eq 0 ]
+    [[ "$output" != "" ]]
+    MINDAYS=(${output//PASS_MIN_DAYS/ }) # get the number from the string
+    [[ "$MINDAYS" -gt 0 ]]
+    run bash -c "grep -E ^[^:]+:[^\!*] /etc/shadow | cut -d: -f1,4"
```

Comment:
This command is not according to the CIS document (`awk -F : '(/^[^:]+:[^!*]/ && $4 < 1){print $1 " " $4}' /etc/shadow`)